omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-HealthEducationPage.xml

```diff
@@ -126,6 +126,19 @@
 							</span>
                     </td>
                 </tr>
+
+                <tr class="enableDisable">
+                    <td>Is Visit On Schedule:</td>
+                    <td id="trigger">
+                        <obs id="msm_final"  conceptId="165134" answerConceptId="160578" answerLabel="Is Visit On Schedule" class="vertical-layout-options"/>                    
```

Comment:
@semujju The right Concept for this field is 90069 but using that will through a 
`java.lang.RuntimeException: Boolean with style = null not yet implemented (concept = 90069)`